app/models/and_joined_facet_subscriber_list.rb

```diff
@@ -0,0 +1,3 @@
+class AndJoinedFacetSubscriberList < SubscriberList
```

Comment:
Is this class necessary? I thought we were just doing Or-Joined lists? PR description doesn't mention the And-Joined list either 🤔

Reply:
I _thought_ it might make it clearer which lists were doing what but happy to remove it